src/utils/stickybits.js

```diff
@@ -1,41 +1,25 @@
 class Stickybits {
   constructor (target) {
-    this.props = {
-      stickyBitStickyOffset: 0,
-      verticalPosition: 'top',
-    }
-    this.target = target
-
-    this.props.positionVal = this.definePosition() || 'fixed'
+    this.target = target;
 
-    const {
-      positionVal,
-      verticalPosition,
-      stickyBitStickyOffset,
-    } = this.props
-    const verticalPositionStyle = verticalPosition === 'top' && `${stickyBitStickyOffset}px`
-    const positionStyle = positionVal !== 'fixed' ? positionVal : ''
-
-    // set vertical position
-    this.target.style[verticalPosition] = verticalPositionStyle
-    this.target.style.position = positionStyle
-  }
+    this.target.style.top = '0px';
+    this.target.style.position = 'sticky -o-sticky -webkit-sticky -moz-sticky -ms-sticky';
 
-  definePosition () {
-    let stickyProp;
-    const prefix = ['', '-o-', '-webkit-', '-moz-', '-ms-']
-    const test = document.head.style
-    for (let i = 0; i < prefix.length; i += 1) {
-      test.position = `${prefix[i]}sticky`
+    if (navigator.appName == 'Netscape') {
+      const ua = navigator.userAgent;
+      const re  = new RegExp("Trident/.*rv:([0-9]{1,}[\\.0-9]{0,})");
+      if (re.exec(ua) != null) {
+        const rv = parseInt( RegExp.$1 );
+        if (rv === 11) {
+          this.target.style.position = 'fixed';
```

Comment:
This doesn't seem correct at first glance. it should only become fixed upon the ad being scrolled to the top of the viewport otherwise the ad will always be stuck to the top of the page

Reply:
hmmm - stickybits wasn't working here, and this other library I found (stickyfill) uses functionality pretty close to scrollMonitor. should I roll a miniature scrollMonitor for ie11 here?